source/write/delete.txt

```diff
@@ -0,0 +1,180 @@
+.. _ruby-write-delete:
+
+================
+Delete Documents
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: remove, drop, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-short+} to remove
+documents from a MongoDB collection by performing **delete operations**.
+
+A delete operation removes one or more documents from a MongoDB collection.
+You can perform a delete operation by using the ``delete_one`` or
+``delete_many`` methods.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your {+language+} application, create a ``MongoClient`` that connects to an Atlas cluster
+and assign the following values to your ``database`` and ``collection`` variables:
+
+.. literalinclude:: /includes/write/delete.rb
+    :language: ruby
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Delete Operations
+-----------------
+
+You can perform delete operations in MongoDB by using the following methods:
+
+- ``delete_one``, which deletes *the first document* that matches the search criteria
+- ``delete_many``, which deletes *all documents* that match the search criteria
+
+Each delete method requires a **query filter** document, which specifies the
```

Comment:
I understand that MongoDB describes these objects as "documents" (because underneath, they are communicated to the server as BSON documents), but it sounds odd to describe a parameter as a "document" in Ruby. Technically, the parameters are Hash instances, but I don't think that needs to be explicitly stated here, as the subsequent examples make it clear what the expectations are.

```suggestion
Each delete method requires a **query filter** parameter, which specifies the
```